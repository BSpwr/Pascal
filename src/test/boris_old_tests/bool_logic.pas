program BoolLogicTest;
var
    val: boolean = true;
    val2: boolean = false;
begin

    writeln('Should be true: ', val or val2);
    writeln('Should be true: ', val || val2);
    writeln('Should be false: ', val and val2);
    writeln('Should be false: ', val && val2);

end.