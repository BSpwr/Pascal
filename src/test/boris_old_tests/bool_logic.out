Should be true: true
Should be true: true
Should be false: false
Should be false: false
